lib/timeline/component/LineGraph.js

```diff
@@ -471,18 +473,15 @@ LineGraph.prototype._updateAllGroupData = function (ids, groupIds) {
           }
         }
       }
-    }
+    } */
 
     //Now insert data into the arrays.
-    for (i = 0; i < items.length; i++) {
```

Comment:
oh, the original code is pretty bad :-(

Reply:
it seemed to be reusing item/groupId from above.